### PR TITLE
Added dockerfile to work in docker environment.

### PR DESCRIPTION
**Pull request by civanescu**
Added dockerfile if anyone want to work in docker environment!

### CODE REVIEW

File: docker-image/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	; \
+	rm -rf /var/lib/apt/lists/*
+
+ENV CONDA_DIR /opt/conda
+
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
+     /bin/bash ~/miniconda.sh -b -p /opt/conda
+
+ENV PATH=$CONDA_DIR/bin:$PATH
+RUN conda install -y -c anaconda python=3.8 \
```
Comment (awvwgk) [inline docker-image/Dockerfile:16]: Why do you downgrade Python to 3.8 in the base environment?

File: docker-image/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    && conda config --add channels conda-forge \
+    && conda install -y -c conda-forge xtb-python qcelemental ase cffi
+
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
```
Comment (awvwgk) [inline docker-image/Dockerfile:21]: Why is this needed? You already installed xtb-python from cf with explicit dependencies, this step should be a no-op if the previous installation worked properly.

**Review by awvwgk — CHANGES_REQUESTED** [review]
Currently, this dockerfile will pull the latest version from cf, rather than building the project in the container. Therefore, I don't see much use of having it as part of the project to be honest.

If this the purpose of the container is to install any xtb python version, then it doesn't seem to be much related to the development of this project. It could be more useful if it provides a reproducible environment for installing the latest source.

File: docker-image/Dockerfile (hunk 2 above)
Comment (civanescu) [reply docker-image/Dockerfile:21]: yeah sorry, the dockerfile was part of a bigger schema of servers in a docker-compose where I had other requirements, probably this shouldn't be here

File: docker-image/Dockerfile (hunk 1 above)
Comment (civanescu) [reply docker-image/Dockerfile:16]: Because appeared to me that you were using 3.8, it was one of the trickiest thing to do.

**Comment by civanescu** [comment]
I built the dockerfile because i was trying to build a docker-compose with multiple servers, i didn't wanted to recompile the source, just to do the conda install and so on... 
P.S. it's your project, you can cancel de PR :).
